lib/api/src/entities/value.rs

```diff
@@ -253,6 +303,7 @@ impl std::fmt::Display for Value {
                 Self::I64(v) => v.to_string(),
                 Self::F32(v) => v.to_string(),
                 Self::F64(v) => v.to_string(),
+                Self::ExceptionRef(_) => "exceptionref".to_string(),
```

Comment:
I think this is commonly known as `exnref`

Reply:
[e04048f](https://github.com/wasmerio/wasmer/pull/5347/commits/e04048f44229e24c4c3f64e524101f5f795c8cdb)